CMakeLists.txt

```diff
@@ -496,27 +496,88 @@ hpx_option(HPX_WITH_PARCELPORT_MPI_MULTITHREADED BOOL
   ON CATEGORY "Parcelport" ADVANCED)
 
 ## Profiling related build options
-hpx_option(HPX_WITH_APEX BOOL "Enable APEX instrumentation support." OFF CATEGORY "Profiling")
+hpx_option(HPX_WITH_APEX BOOL
+  "Enable APEX instrumentation support." OFF CATEGORY "Profiling")
 if(HPX_WITH_APEX)
   hpx_add_config_define(HPX_HAVE_APEX)
 endif()
-hpx_option(HPX_WITH_PAPI BOOL "Enable the PAPI based performance counter." OFF CATEGORY "Profiling")
+hpx_option(HPX_WITH_PAPI BOOL
+  "Enable the PAPI based performance counter." OFF CATEGORY "Profiling")
 if(HPX_WITH_PAPI)
   hpx_add_config_define(HPX_HAVE_PAPI)
 endif()
-hpx_option(HPX_WITH_TAU BOOL "Enable TAU profiling support." OFF CATEGORY "Profiling")
+hpx_option(HPX_WITH_TAU BOOL
+  "Enable TAU profiling support." OFF CATEGORY "Profiling")
 if(HPX_WITH_TAU)
   hpx_add_config_define(HPX_HAVE_TAU)
 endif()
-hpx_option(HPX_WITH_GOOGLE_PERFTOOLS BOOL "Enable Google Perftools instrumentation support." OFF CATEGORY "Profiling")
+hpx_option(HPX_WITH_GOOGLE_PERFTOOLS BOOL
+  "Enable Google Perftools instrumentation support." OFF CATEGORY "Profiling")
 if(HPX_WITH_GOOGLE_PERFTOOLS)
   hpx_add_config_define(HPX_HAVE_GOOGLE_PERFTOOLS)
 endif()
-hpx_option(HPX_WITH_ITTNOTIFY BOOL "Enable Amplifier (ITT) instrumentation support." OFF CATEGORY "Profiling")
+hpx_option(HPX_WITH_ITTNOTIFY BOOL
+  "Enable Amplifier (ITT) instrumentation support." OFF CATEGORY "Profiling")
 #if(HPX_WITH_ITTNOTIFY)
   #hpx_add_config_define(HPX_HAVE_ITTNOTIFY)
 #endif()
 
+## External libraries/frameworks used by sme of the examples and benchmarks
+hpx_option(HPX_WITH_OPENMP BOOL
```

Comment:
Am 03.06.2016 9:58 nachm. schrieb "Hartmut Kaiser" <notifications@github.com
>:
>
> In CMakeLists.txt:
>
> >  #if(HPX_WITH_ITTNOTIFY)
> >    #hpx_add_config_define(HPX_HAVE_ITTNOTIFY)
> >  #endif()
> >
> > +## External libraries/frameworks used by sme of the examples and
benchmarks
> > +hpx_option(HPX_WITH_OPENMP BOOL
>
> I introduced those options in response to the question we've got at CERN
where somebody asked why we would require TBB, qthreads, etc. as those are
being reported as not available...

How about naming the option HPX_WITH_EXAMPLES_OPENMP?

>
> —
> You are receiving this because you commented.
>
> Reply to this email directly, view it on GitHub, or mute the thread.